README.md

```diff
@@ -17,13 +17,30 @@ make
 
 ## Settings
 ```
- PEAK_TARGET=dgemm_,dgemv_        # functions that will be profiled
- PEAK_COST=10                     # Upper limit of profiling cost. The monitoring process will detach if the total profiling cost exceeds this value.  
+PEAK_TARGET=dgemm_,dgemv_        # functions that will be profiled
+PEAK_COST=10                     # Upper limit of profiling cost. The monitoring process will detach if the total profiling cost exceeds this value.  
```

Comment:
It should be "Upper limit of profiling cost in seconds". We need to claim the unit here.